src/core.jl

```diff
@@ -210,72 +227,102 @@ function _push!(n, x, onerror=print_error)
     end
     nothing
 end
-_push!(::Void, x, onerror=print_error) = nothing
-
-const timestep = Ref{Int}(0)
+async_push!(::Void, x, onerror=print_error) = nothing
```

Comment:
Actually, that line won't run because we only define `push!(n::Signal, ...`. do you think we should define `push!(::Void, ... ) = nothing` ?